### DISCUSSION TITLE
Integrate Tailwindcss

### DISCUSSION THREAD

**Discussion by byhbt** [nimblehq/elixir-templates#272] [RFCs]
## Why

Tailwindcss

## Who Benefits?

Elixir developer

**Comment by byhbt** [comment]
The original idea comes from the issue: https://github.com/nimblehq/elixir-templates/issues/66

![image](https://user-images.githubusercontent.com/948856/192507034-e3865c96-5b69-4966-8f04-792ba360f43d.png)

**Comment by andyduong1920** [comment]
@byhbt as Tailwindcss is part of Phoenix 1.7 now, do you think we still need this?
